lib/traits/RenderableView.js

```diff
@@ -45,6 +45,7 @@ var RenderableView = _.extend({},
             }
 
             this.onDOM();
+            this.trigger("on-dom");
```

Comment:
any reason you chose to trigger the event before the isInDOM state changed? Can you see any use case where this would be a problem?